assets/icons.tsx

```diff
@@ -684,3 +684,21 @@ export const SearchClockIcon = () => {
     </svg>
   );
 };
+
+export const LogoutIcon = () => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      fill='none'
+      viewBox='0 0 24 24'
+      strokeWidth='1.5'
+      stroke='currentColor'
+      className='w-6 h-6'>
+      <path
+        stroke-linecap='round'
```

Comment:
Convert `stroke-linecap` this type of attributes to `strokeLinecap` because a dash into attribute name is not valid in react, it will produce warning while building.